uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/themes/GridRendererTheme.java

```diff
@@ -31,6 +31,10 @@
  */
 public interface GridRendererTheme {
 
+    String HIGHLIGHTED_CELL_BACKGROUND = "#F9D67A";
+
+    String HIGHLIGHTED_CELL_STROKE = "#F5C12E";
```

Comment:
I think we should move these two constants into:
`org.uberfire.ext.wires.core.grids.client.widget.grid.renderers.themes.impl.KIEColours`